Makefile

```diff
@@ -31,8 +36,24 @@ endif
 
 COVER_THRESHOLD := $(shell grep '^name: cover' .github/workflows/ci-go-cover.yml | cut -c13-)
 
+
+define MOCK_template
+mock_$(1): $(1)
+	$$(MOCKGEN) -source=$$< -destination=cocli/cmd/mocks/$$$$(basename $$@) -package=$$(MOCKPKG)
+endef
+
+$(foreach m,$(INTERFACES),$(eval $(call MOCK_template,$(m))))
+MOCK_FILES := $(foreach m,$(INTERFACES),$(join mock_,$(m)))
```

Comment:
just a comment: this looks extremely heavyweight for just one interface.

Reply:
As discussed would prefer to retain for future more mock generation!